src/lib/components/Tooltip/index.tsx

```diff
@@ -76,40 +79,34 @@ export const Tooltip: FC<TooltipProps> = ({
 
   return (
     <>
-      <div className="w-fit" {...getReferenceProps({ ref: reference })} data-testid="tooltip-target">
+      <div className={theme.target} {...getReferenceProps({ ref: reference })} data-testid="tooltip-target">
         {children}
       </div>
       <div
         data-testid="tooltip"
         {...getFloatingProps({
-          className: classNames(
-            'absolute inline-block rounded-lg py-2 px-3 text-sm font-medium shadow-sm',
-            animation !== false && `transition-opacity ${animation}`,
-            {
-              'invisible opacity-0': !open,
-              'bg-gray-900 text-white dark:bg-gray-700': style === 'dark',
-              'border border-gray-200 bg-white text-gray-900': style === 'light',
-              'border border-gray-200 bg-white text-gray-900 dark:border-none dark:bg-gray-700 dark:text-white':
-                style === 'auto',
-            },
-            className,
-          ),
+          className: classNames(theme.base, animation !== false && `${theme.animation} ${animation}`, {
```

Comment:
Is !== false needed?

Reply:
Nope! Well noticed, I was just replacing the styles to add the theme, didn't review the existing code base.